test/apps/iodemo/io_demo.cc

```diff
@@ -87,6 +87,7 @@ typedef struct {
     bool                     use_epoll;
     ucs_memory_type_t        memory_type;
     unsigned                 progress_count;
+    struct sockaddr_storage  local_addr;
```

Comment:
const char* local_addr
To align with server address type. Just convert it to struct sockaddr_storage when using it.

Reply:
fixed